### ISSUE TITLE
Can't declare classhash

### ISSUE THREAD

**Issue by Th0rgal** [xJonathanLEI/starkli#85]
Hey, I'm encountering a persistent error when trying to declare a class hash using Starkli, despite setting up the environment variables correctly and confirming the validity of the keystore. This issue happens no matter which RPC or network settings I use.

**Steps to Reproduce:**
1. First, I set up the environment variables for Starkli like so:
```
   export STARKNET_ACCOUNT="~/.starkli-wallets/sepolia/th0rgal.json"
   export STARKNET_RPC="https://sepolia.rpc.starknet.id/"
   export STARKNET_KEYSTORE="~/.starkli-wallets/sepolia/th0rgal.keystore.json"
```
2. Then, I run the command to declare the contract class:
```
   starkli declare target/release/composable_multicall_ComposableMulticall.compiled_contract_class.json
```

**Expected Result:**
The class hash should be declared smoothly without any errors.

**Actual Result:**
Unfortunately, the declaration process fails, suggesting there might be an issue with Starkli or perhaps some aspect of my setup.

**Additional Information:**
- I've double-checked my environment setup and keystore, both are set up correctly.
- I've also tried using different RPCs and specifying different networks, but the problem persists.

**Configuration Files:**
- Here’s the content of my account JSON file:
```
  {
    "version": 1,
    "variant": {
      "type": "argent",
      "version": 1,
      "owner": "0x53127b46c7f449bb61ee31f29b4762c823889c81dae48b4037f7ba0948398e6",
      "guardian": "0x0"
    },
    "deployment": {
      "status": "deployed",
      "class_hash": "0x29927c8af6bccf3f6fda035981e765a7bdbf18a2dc0d630494f8758aa908e2b",
      "address": "0x403c80a49f16ed8ecf751f4b3ad62cc8f85ebeb2d40dc3b4377a089b438995d"
    }
  }
```

**Comment by xJonathanLEI** [comment]
> Unfortunately, the declaration process fails, suggesting there might be an issue with Starkli or perhaps some aspect of my setup.

What exactly is the failure though?

**Comment by Th0rgal** [comment]
> ```
>    starkli declare target/release/composable_multicall_ComposableMulticall.compiled_contract_class.json
> ```

Sorry I forgot the most important part, I just get:
```
Error: No such file or directory (os error 2)
```

**Comment by xJonathanLEI** [comment]
Thanks for reporting. Found a bug that's presumably already fixed but turns out it still hangs around. Fixed in https://github.com/xJonathanLEI/starkli/commit/53ffb435fff917a245e3ef5e417a73299a742730. Releasing soon.

Meanwhile, you can work around this issue by using full path for `STARKNET_ACCOUNT` instead of including `~`.

**Comment by Th0rgal** [comment]
Awesome. Thank you

**Comment by xJonathanLEI** [comment]
The fix has now been released with [v0.2.9](https://github.com/xJonathanLEI/starkli/releases/tag/v0.2.9).
